src/schnetpack/configs/model/representation/painn.yaml

```diff
@@ -1,5 +1,6 @@
 defaults:
   - radial_basis: gaussian
+  - nuclear_embedding: complex
```

Comment:
remove this and leave default value as it is

Reply:
I agree, done in 0f6fc8bae9a950c8f6130b92eb013b4d6f01c95f